### ISSUE TITLE
(Java Agent) - Migrate the Vault access for Team services to service-permissions

### ISSUE THREAD

**Issue by kford-newrelic** [newrelic/newrelic-java-agent#1492]
### Description
CSI is in the process of migrating manually created Vault AppRoles and Tokens to an automated process.

We have identified that Java Agent team is currently using Vault AppRoles or Tokens that need to be migrated to the new CSI service permissions.

Here is the list of affected **Tokens**:

Vault US
java-agent/integration/performance-tests-ro
If any of these services are not part of your team, please inform us. 

[In this documentation](https://pages.datanerd.us/site-engineering/nr-platform-docs/access_to_systems/automated_service_access/vault.html), y'all will find enough information to decide the appropriate type of access for your services.

The documentation also covers the benefits and implications of each option.

### Acceptance Criteria
Team tokens and AppRoles are defined in https://source.datanerd.us/csi-shared/service-permissions
Services are updated to use the new credentials
Ensure that the old credentials are ready to be removed from CSI on **29th of September** (Note : this date will not be met)

If you have any questions or need further assistance, please let us know.

### Design Consideration/Limitations
N/A

### Dependencies
N/A

### Additional context
N/A

**Comment by jasonjkeller** [comment]
This was related to this old Jira which is no longer accessible: [JAVA-3287: Update Google docs spreadsheet reporter to use vault](https://newrelic.atlassian.net/browse/JAVA-3287?focusedCommentId=302665&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-302665)

Context from that Jira:

```
In summary:
There are two new vault policies now:
teams/java-agent/integration/performance-tests
teams/java-agent/integration/performance-tests-ro

The polices control access to the following key space: containers/teams/java-agent/integration/performance-tests/*

The first policy allows specific ldap users with chicago vpn access to read and write keys and the second is read only access. LDAP users on the policy with chicago vpn access can read / write values. Non-ldap users can use a specially generated vault token which I have stored on hans and frau in environment variables. However, it still requires being on chicago vpn which hans and frau aren't :(

"PDX Jenkins cannot access Vault. We don't have a solution in place for build jobs running outside CHI. The only option, which isn't great, is to use the credential storage module...
```

TL;DR

Vault was used to store secrets for an old performance test to allow it to publish results to a Google sheet. https://github.com/newrelic/java-agent-integration-tests/blob/0572e733fff40d2e14e1cb129e13c1d8b2492641/tests/java/perf/Jenkinsfile#L443

This performance test hasn't been run in a number of years (since 2019) and the google-sheets-reporter no longer exists.

I think these vault policies can be deleted at this point.

**Comment by jasonjkeller** [comment]
Note for CSI: Any vault tokens associated with following vaults are no longer used or needed. Please feel free to delete them when you see fit.

```
teams/java-agent/integration/performance-tests
teams/java-agent/integration/performance-tests-ro
```
